README.adoc

```diff
@@ -141,6 +142,24 @@ Flags:
       --property stringArray   Add a source property in the form of "<key>=<value>"
 ----
 
+==== `binding delete`
+
+----
+Delete Kamelet binding by its name.
+
+Usage:
+  kn-source-kamelet binding delete [flags]
```

Comment:
Regenerate readme? (same as the other PR comment :))

Reply:
👍